src/js/pre-need/config/form.jsx

```diff
@@ -498,12 +498,10 @@ const formConfig = {
                   name: _.merge(fullNameUI, {
                     'ui:title': 'Name of deceased'
                   }),
-                  'view:cemeteryNumber': {
-                    'ui:title': 'VA national cemetery where they’re buried'
-                    // TODO: Create widget with validation message...
-                    // It should map hundreds of cemetery numbers to names.
-                    // 'ui:widget': CemeteryNumberWidget
-                  }
+                  cemeteryNumber: autosuggest.uiSchema(
+                    'VA National Cemetery where they’re buried',
```

Comment:
Should this ("national cemetery") be lowercase?

Reply:
Yep, it should.